app/models.py

```diff
@@ -288,6 +288,19 @@ class Config:
         }
 
 
+class QuizCreationResponse(BaseModel):
+    """Model for the response of a request that creates a quiz"""
+
+    quiz_id: str
```

Comment:
just `id` should be enough no?
It will be the id of a quiz that much is understood

Reply:
let's leave it as `quiz_id` only, because creator/user will be interacting with it and it may be interpreted as `user_id`. I think in session model too we use `quiz_id`

Reply:
also in `router/quizzes.py`,  `@router.get("/{quiz_id})` we mention as `quiz_id`